docs/openapi/components/schemas/common/AgGradeInspectionResult.yml

```diff
@@ -0,0 +1,45 @@
+$linkedData:
+  term: AgGradeInspectionResult
+  '@id': https://w3id.org/traceability#AgGradeInspectionResult
+title: Agriculture Grade Inspection Result
+description: Information on the grade assigned to an inspected lot.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionResult
+      - type: string
+        const:
+          - AgGradeInspectionResult
+  gradeInspected:
+    title: Grade Inspected
+    description: Grade the lot was inspected for, even if the requirements were not met. When there is no U.S. grade standard for a product, and a quality and condition inspection is performed, enter "No established U.S. Grade".
+    type: string
+    $linkedData:
+      term: gradeInspected
+      '@id': https://w3id.org/traceability#gradeInspected
```

Comment:
We don't currently define this term. https://w3id.org/traceability#gradeInspected

Reply:
I think I'm less confused now on the role of @id and the issue with including undefined terms, but I still can't find anything in schema.org (or gs1.org) appropriate for a food grade string.

Would you accept leaving this as is, for lack of a better option?